scripts/run-framework/agief_experiment/experiment.py

```diff
@@ -170,6 +173,13 @@ def run_parameterset(self, compute_node, cloud, args, entity_filepath, data_file
         with open(info_filepath, 'w') as data:
             data.write(info)
 
+        # Silently remove older log file if exists
+        log_filepath = self.experiment_utils.runpath(self.LOG_FILENAME)
+        if compute_node.remote():
+             utils.remote_run(compute_node.host_node, 'rm ' + log_filepath)
```

Comment:
This has 1 additional space indented. Not sure what effect this has.
I've corrected it, but confirm this doesn't change functionality (python is sensitive)